support /dev/ttyACMx devices natively
upstream bug report pending: http://bugzilla.qbang.org/show_bug.cgi?id=139
http://bugs.gentoo.org/show_bug.cgi?id=301126
Signed-off: Thilo Bangert <bangert@gentoo.org>

--- src/gnu/io/RXTXCommDriver.java.orig	2010-01-15 17:30:46.000000000 +0100
+++ src/gnu/io/RXTXCommDriver.java	2010-01-15 17:31:56.000000000 +0100
@@ -519,6 +519,7 @@
 					if(osName.equals("Linux"))
 					{
 						String[] Temp = {
+						"ttyACM",       // linux CDC ACM devices
 						"ttyS", // linux Serial Ports
 						"ttySA", // for the IPAQs
 						"ttyUSB" // for USB frobs
